.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release Gem to rubygems.org
+
+on:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      id-token: write
+
+    environment: rubygems.org
+
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@0cde4689ba33c09f1b890c1725572ad96751a3fc # v1.178.0
+        with:
+          bundler-cache: true
+          ruby-version: ruby
+
+      # Release
+      - uses: rubygems/release-gem@612653d273a73bdae1df8453e090060bb4db5f31 # v1
```

Comment:
I've added this to the actions whitelist!

Looks like it uses `rake release` internally so may need a `Bundler::GemHelper.install_tasks` or whatever in the Rakefile.

Reply:
Hmm -- I adapted this from `rubygems-await`, which doesn't seem to use that in their Rakefile: https://github.com/segiddins/rubygems-await/blob/main/Rakefile

(But I'm fully cargo-culting this, so maybe I missed an equivalent.)

Reply:
`require "bundler/gem_tasks"` is the magic line that does it for you:

https://github.com/rubygems/rubygems/blob/master/bundler/lib/bundler/gem_tasks.rb#L7